fhir-persistence-schema/src/main/resources/db2/add_any_resource.sql

```diff
@@ -69,9 +69,9 @@ BEGIN
   THEN
     VALUES NEXT VALUE FOR {{SCHEMA_NAME}}.fhir_sequence INTO v_logical_resource_id;
     PREPARE stmt FROM
-       'INSERT INTO ' || v_schema_name || '.logical_resources (mt_id, logical_resource_id, resource_type_id, logical_id) '
-    || '     VALUES (?, ?, ?, ?)';
-    EXECUTE stmt USING {{ADMIN_SCHEMA_NAME}}.sv_tenant_id, v_logical_resource_id, v_resource_type_id, p_logical_id;
+       'INSERT INTO ' || v_schema_name || '.logical_resources (mt_id, logical_resource_id, resource_type_id, logical_id, reindex_tstamp) '
+    || '     VALUES (?, ?, ?, ?, ?)';
+    EXECUTE stmt USING {{ADMIN_SCHEMA_NAME}}.sv_tenant_id, v_logical_resource_id, v_resource_type_id, p_logical_id, '1970-01-01-00.00.00.0';
```

Comment:
should we add it with the current/"now" timestamp instead?

Reply:
I think it's OK as-is, but I'm curious as to the logic.

Reply:
See previous comment about not being able to use NULL for performance reasons, but want a way to indicate the resource had never been reindexed.